### ISSUE TITLE
en:nutrition-value-over-105-fruits-vegetables-nuts-estimate-from-ingredients: false positive

### ISSUE THREAD

**Issue by CharlesNepote** [openfoodfacts/openfoodfacts-server#7816]
### Describe the bug

Sometimes fruit estimation is higher than 105 while it shouldn't be.
Eg. https://world.openfoodfacts.org/cgi/product.pl?type=edit&code=3038354191904#ingredients
![image](https://user-images.githubusercontent.com/562352/206651683-262ed27e-5606-4819-afd9-f4c44cd67751.png)

The json file displays 174.5 in this example:
![image](https://user-images.githubusercontent.com/562352/206652004-7488e107-30fe-4a7a-8df5-4fdb8058e56c.png)

### To Reproduce

...

### Expected behavior

...

### Type of device

Browser

**Comment by stephanegigandet** [comment]
The issue is from ingredient parsing, we turn "tomato pulp and tomato puree (72%)" into "tomato pulp (72%), tomato puree (72%)".

One solution could be to make it a composite ingredient instead "tomato pulp and tomato puree (72%) (tomato pulp, tomato puree)".

**Comment by benbenben2** [comment]
> The issue is from ingredient parsing, we turn "tomato pulp and tomato puree (72%)" into "tomato pulp (72%), tomato puree (72%)". One solution could be to make it a composite ingredient instead "tomato pulp and tomato puree (72%) (tomato pulp, tomato puree)".

Yes, but if your input is  "tomato pulp and tomato puree (72%) (tomato 95%, water 5%)"  then, your output would be something like "tomato pulp and tomato puree (72%) (tomato pulp, tomato puree) (tomato 95%, water 5%)" not sure it is good.

I would rather simply not split it in two if there is percent. 
(+) we will not have problem if it is compound/preparation
(+) it solves percentage analysis
(-) we have to add the whole compound in the taxonomy
